pkgs/by-name/lo/localproxy/package.nix

```
@@ -31,6 +31,11 @@ stdenv.mkDerivation (finalAttrs: {
       url = "https://github.com/aws-samples/aws-iot-securetunneling-localproxy/commit/de8779630d14e4f4969c9b171d826acfa847822b.patch";
       hash = "sha256-11k6mRvCx72+5G/5LZZx2qnx10yfKpcAZofn8t8BD3E=";
     })
+    (fetchpatch {
+      name = "boost187.patch";
+      url = "https://github.com/aws-samples/aws-iot-securetunneling-localproxy/commit/7cb3478c26d09ed2056200bfbbd44e14e17e23d8.patch";
```

Since the PR was squash merged, that commit is dangling. 

```suggestion
      url = "https://github.com/aws-samples/aws-iot-securetunneling-localproxy/commit/12022770e89c6787c3eda4ca01a7cedaf2affa92.patch";
```
id the ci change causes problem, please add the file(s) to excludes.